### ISSUE TITLE
Make the Android UI pretty

### ISSUE THREAD

**Issue by paulrouget** [servo/servo#21403]
As of now, it looks like this:

<img width="406" alt="screen shot 2018-08-14 at 09 50 33" src="https://user-images.githubusercontent.com/373579/44078722-87f2bd00-9fa7-11e8-998b-7c3f61991b8a.png">

There are plenty of ways to make this look better. Maybe the buttons should be under a drop down menu. We should use icons instead of text. The toolbar should be merged with the UI top bar. Etc.

To make changes to the UI, all that is needed is Android Studio. Open support/android/apk, and then select `servoapp/res/layout/activity_main.xml`:

<img width="1136" alt="screen shot 2018-08-14 at 10 04 08" src="https://user-images.githubusercontent.com/373579/44079470-6d89238a-9fa9-11e8-8227-381f7f0f206d.png">

A lot of things can be done from this Design tool.

It's likely that some changes to `MainActivity.java` will be required too.

For now, it's still necessary to [manually compile Servo](https://github.com/servo/servo/wiki/Building-for-Android) but we hope to make the process a lot easier: https://github.com/servo/servo/issues/21402

If anyone is interested to work on this, I'd be happy to help.

**Comment by paulrouget** [comment]
See: https://github.com/servo/servo/wiki/Building-for-Android#working-on-the-user-interface-without-building-servo

**Comment by paulrouget** [comment]
Even the UI assets? Like back and forward icons?

**Comment by jdm** [comment]
Yes. We do not want our application to use any of Firefox's assets.

**Comment by wayling** [comment]
i have interested to do but it has any suggestions?

**Comment by paulrouget** [comment]
> any suggestions

Using icons instead of text could be a good start.

**Comment by Gae24** [comment]
With #43795 this can be considered done, the search bar could still use some work, though.
